src/WebJobs.Script/Description/DotNet/FunctionAssemblyLoadContext.cs

```diff
@@ -40,13 +42,42 @@ public FunctionAssemblyLoadContext(string basePath)
 
         public static FunctionAssemblyLoadContext Shared => _defaultContext.Value;
 
+        private static Dictionary<string, ResolutionPolicyEvaluator> InitializeLoadPolyEvaluators()
+        {
+            return new Dictionary<string, ResolutionPolicyEvaluator>
+            {
+                { "minorMatchOrLower", IsMinorMatchOrLowerPolicyEvaluator },
+                { "runtimeVersion", RuntimeVersionPolicyEvaluator }
+            };
+        }
+
+        private static bool RuntimeVersionPolicyEvaluator(AssemblyName arg1, Assembly arg2) => true;
+
+        private static bool IsMinorMatchOrLowerPolicyEvaluator(AssemblyName requestedAssembly, Assembly runtimeAssembly)
```

Comment:
Would be good to put some code comments in here for all of this, detailing how it works, what it is doing